appengine/websockets/test/index.test.js

```diff
@@ -15,18 +15,24 @@
 'use strict';
 
 const assert = require('assert');
+const path = require('path');
+const app = require(path.join(__dirname, '../', 'app.js'));
```

Comment:
Nit: would `path.dirname(__dirname)` be cleaner/more resilient here?

Reply:
Done ;)